pkg/solana/chainwriter/ccip_example_config.go

```diff
@@ -1,28 +1,42 @@
 package chainwriter
 
 import (
+	"context"
+	"fmt"
+	"reflect"
+
+	ag_binary "github.com/gagliardetto/binary"
 	"github.com/gagliardetto/solana-go"
 )
 
+const registryAddress = "4Nn9dsYBcSTzRbK9hg9kzCUdrCSkMZq1UR6Vw1Tkaf6A"
+
+// Should match type here: https://github.com/smartcontractkit/chainlink-ccip/blob/6bb9583526ce7c281005d66ffdf4a5300fc8ddc9/chains/solana/gobindings/ccip_router/accounts.go#L707-L713
```

Comment:
can't you import that file? I thought that it was the idea from exposing those in the public repository

Reply:
Yup, good point. Fixed